docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -117,31 +122,55 @@ There are several versions of CUDA on Discovery, as listed below.::
   cuda/11.1
   cuda/11.2
   cuda/11.3
+  cuda/11.4
+
+You can always use the ``module avail`` command to check for the latest software versions on Discovery as well. To see details on a specific CUDA toolkit version, use ``module show``. For example, ``module show cuda/11.4``.
 
-You can always use the ``module avail`` command to check for the latest software versions on Discovery as well.
+To add CUDA to your path use ``module load``. For example, type ``module load cuda/11.4`` to load version 11.4 to your path.
 
-To add CUDA to your path use ``module load``. For example, type ``module load cuda/10.0`` to load version 10 to your path.
+Use the command ``nvidia-smi`` (NVIDIA System Management Interface) inside a GPU node to get the CUDA driver information and monitor the GPU device.
 
 Using GPUs with PyTorch
 ========================
 You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery.
 
-The following is an example of using a conda virtual environment with PyTorch for CUDA version 11.1. Make sure that you are on a GPU node before loading the environment::
+The following examples demonstrate how to build PyTorch inside a conda virtual environment for CUDA version 11.3. Make sure that you are on a GPU node before loading the environment.
 
-  module load cuda/11.1
+Lightweight installation::
+  
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  module load cuda/11.3
   module load anaconda3/2021.11
   conda create --name pytorch_env python=3.7 -y
   source activate pytorch_env
-  conda install pytorch==1.8.0 torchvision==0.9.0 torchaudio==0.8.0 cudatoolkit=11.1 -c pytorch -c conda-forge -y
+  conda install pytorch==1.9.0 torchvision==0.10.0 torchaudio==0.9.0 cudatoolkit=11.3 -c pytorch -c conda-forge -y
+  python -c'import torch; print(torch.cuda.is_available())'
+
+Heavyweight installation (with Anaconda libraries)::
+
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  module load cuda/11.3
+  module load anaconda3/2021.11
+  conda create --name pytorch_env python=3.7 anaconda -y
+  source activate pytorch_env
+  conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch -y
   python -c'import torch; print(torch.cuda.is_available())'
 
-As the latest version of Pytorch often depends the newst CUDA avaialble, please refer to the Pytorch documentation page for the installation instructions: https://pytorch.org/. 
+As the latest version of PyTorch often depends the newst CUDA avaialble, please refer to the PyTorch documentation page for the installation instructions: https://pytorch.org/. 
```

Comment:
"...often depends on the newest CUDA available..."

Reply:
This hasn't been incorporated yet. There are typos in the current statement and if you inserted the suggestions then they are not showing up on the draft.